automatic/waterfox/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,27 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$packageName = 'waterfox'
+
+$uninstalled = $false
```

Comment:
I know it's not used anywhere, however it's part of the default template if you run `choco new`.
I don't really know if it's perhaps used somehow by choco or one of its helpers.

Maybe @ferventcoder can educate us on why it's in the default template?